ast/src/analyzed/mod.rs

```diff
@@ -631,11 +633,18 @@ impl PublicDeclaration {
     }
 }
 
-#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, JsonSchema)]
-pub struct Identity<Expr> {
+#[derive(
+    Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Clone, Copy, Serialize, Deserialize, JsonSchema,
+)]
+pub struct IdentityId {
     /// The ID is specific to the identity kind.
-    pub id: u64,
+    pub local_id: u64,
```

Comment:
This is the case for all other IDs. Not sure if this justifies calling it `local_id`. Also I think it is confusing. What is it local to?